src/router/index.js

```diff
@@ -76,9 +76,7 @@ router.beforeEach(async (to, from) => {
   if (!store.state.user.user.permissions.includes('read') && to.name !== 'noAuth') {
     return { name: 'noAuth' }
   }
-})
 
-router.beforeResolve((to, from) => {
```

Comment:
Interesting, I would like to understand this change.

Reply:
Discussed in person. When navigating from one workflow to another, the subscription would start using the old workflow name for a short while and then update to the new workflow name. Now the subscription starts using the new workflow name